### ISSUE TITLE
IFC 4.3.2.0. Schema version (already ISO) implementation

### ISSUE THREAD

**Issue by daviddelven** [BIM-Tools/SketchUp-IFC-Manager#57]
Is there some plan to implement the latest official version of IFC Schema IFC 4.3.2.0. ?
Regardless the multiple domain additions, basically from Infrastructure, there are also some updates from Building classes.

**Comment by janbrouwer** [comment]
Yes working on that, it messes up the spatial hierarchy (again) so I need to think that out well
